projects/pgai/pgai/vectorizer/vectorizer.py

```diff
@@ -122,6 +122,62 @@ class Vectorizer:
     schema: str = "ai"
     table: str = "vectorizer"
 
+    async def run(
+        self,
+        db_url: str,
+        features: Features,
+        worker_tracking: WorkerTracking,
+        concurrency: int | None,
+        should_continue_processing_hook: None | Callable[[int, int], bool] = None,
+    ) -> int:
+        """Run this vectorizer with the specified configuration using Worker instances
+
+        Args:
+            db_url: Database connection URL
+            features: Features from database
+            worker_tracking: Tracking instance for worker stats
+            concurrency: Number of concurrent workers
+                (overrides vectorizer config if provided)
+            should_continue_processing_hook: Optional callback to
+                control processing flow
+
+        Returns:
+            Number of items processed
+        """
+        concurrency = concurrency or self.config.processing.concurrency
```

Comment:
Here it is the fallback I mentioned in https://github.com/timescale/pgai/pull/572/files#r2005248136